.github/workflows/vpdq-release.yaml

```diff
@@ -17,24 +17,31 @@ jobs:
   publish:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
         with:
-          python-version: "3.8"
+          python-version: "3.10"
       - name: Install packaging dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install -r requirements.txt
-          python vpdq-release.py -i
-      - name: Package threatexchange
+          python -m pip install -r packaging-requirements.txt
+
+      - name: Package
+        run: |
+          python -m build --sdist
+
+      - name: Ensure packaged build runs
```

Comment:
Maybe move this check to the other test file?

Still this is an important sanity check and doing it before uploading seemed logical. If I had this check in other projects I wouldn't have published broken builds.

Reply:
This step is the one that fails post-diff, I'm not sure how to test it other than creating a temporary copy that runs on contbuild.

I think it's missing a step to install ffmpeg, let me see if I can get a repro